_posts/en/newsletters/2020-06-03-newsletter.md

```diff
@@ -0,0 +1,310 @@
+  or bug-free, but both should be useful to developers who want to
+  quickly compare the sizes of different types of transactions.
+
+## Recently transcribed talks and conversations
+
+*[Bitcoin Transcripts][] is the home for transcripts of technical
+Bitcoin presentations and discussions. In this monthly feature, we
+highlight a selection of the transcripts from the previous month.*
+
+- **LN backups:** Christian Decker presented at Potzblitz on the latest
+  state of LN backups. He discussed the approaches of other
+  implementations such as Eclair and LND before explaining why
+  C-Lightning is using a synchronous database log plugin. Later, he
+  described why LN backups are more complex than onchain backups, the
+  prospects of adding [SIGHASH_NOINPUT][topic sighash_noinput] or
+  `SIGHASH_ANYPREVOUT` to Bitcoin to enable [eltoo-based][topic eltoo]
+  payment channels, and the current modularity of the LN protocol.
+  ([transcript][decker xs], [video][decker vid])
+
+- **Payjoin/P2EP:** Adam Gibson led a discussion at London BitDevs about
+  [payjoin][topic payjoin], a protocol that allows both the sender and
+  receiver of a payment to contribute inputs to the transaction.  This
+  breaks the [common wallet ownership assumption][] and subset sum
+  analysis, improving the privacy of both the sender and the receiver.
+  Gibson went through the history
+  of the concept and described the existing implementations of payjoin
+  in JoinMarket and Samourai before examining details of the recent
+  BTCPay Server implementation. He ended by outlining several different
+  ways a wallet can be fingerprinted, such as the number of signatures
+  required, what timelocks are used, and whether the opt-in
+  Replace-By-Fee ([RBF][topic rbf]) flag is set.  ([transcript][gibson
+  xs], [video][gibson vid])
+
+- **LSAT---your ticket aboard the lightning native web:** Oliver Gugger
+  presented the Lightning Service Authentication Token (LSAT) at
+  Reckless VR in virtual reality. LSAT is a proposed protocol
+  specification combining HTTP, macaroons, and Lightning. It's designed
+  to fulfill the purpose of the HTTP 402 Payment Required response code.
+  Gugger described the authentication flow and the role of macaroons as
+  pseudonymous user authentication. The question and answer session
+  focused on use cases and the benefits of using LSAT, such as enhanced
+  user privacy and improved sign-up experience.
+  ([transcript](https://diyhpl.us/wiki/transcripts/vr-bitcoin/2020-05-16-oliver-gugger-lsat/),
+  [video](https://www.youtube.com/watch?v=IW08RJUpzw0))
+
+- **Sydney meetup discussion:** A number of Bitcoin and LN developers
+  joined this Sydney meetup to discuss topics including: the scalability
+  issues of onboarding millions of LN clients, Rust code integration
+  into Bitcoin Core, dual funding in C-Lightning, and modern soft fork
+  activation. The history of Linux kernel development and segwit
```

Comment:
Modern isn't really the right word here. It's more like "possible soft fork activation methods".